forest/jasmine/traj2stats.py

```diff
@@ -473,21 +474,45 @@ def gps_summaries(
             for pause in pause_array:
                 if places_of_interest is not None:
                     all_place_probs = [0] * len(places_of_interest)
-                    pause_circle = transform_point_to_circle(
-                        pause[0], pause[1], person_point_radius
-                    )
+                    if (
+                        f"{pause[0]}, {pause[1]} - person"
+                        in saved_polygons.keys()
+                    ):
+                        pause_circle = saved_polygons[
+                            f"{pause[0]}, {pause[1]} - person"
+                        ]
+                    else:
+                        pause_circle = transform_point_to_circle(
+                            pause[0], pause[1], person_point_radius
+                        )
+                        saved_polygons[
+                            f"{pause[0]}, {pause[1]} - person"
+                        ] = pause_circle
                     add_to_other = True
                     for j, place in enumerate(places_of_interest):
                         for element_id in ids[place]:
                             if len(locations[element_id]) == 1:
-                                location_circle = transform_point_to_circle(
-                                    locations[element_id][0][0],
-                                    locations[element_id][0][1],
-                                    place_point_radius,
-                                )
+                                loc_lat = locations[element_id][0][0]
+                                loc_lon = locations[element_id][0][1]
+                                if (
+                                    f"{loc_lat}, {loc_lon} - place"
```

Comment:
Could you make this a variable? I think it would make the code less repetitive and more readable.